iceoryx_posh/include/iceoryx_posh/runtime/service_discovery.hpp

```diff
@@ -52,6 +52,7 @@ class ServiceDiscovery
     /// @param[in] service service string to search for, a nullopt corresponds to a wildcard
```

Comment:
We need to mention in the `brief` section that we match all services with the messaging pattern we provide.